lib/src/utils/cached_profile_information.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:matrix/matrix_api_lite.dart';
+
+class CachedProfileInformation extends ProfileInformation {
```

Comment:
can we also add `userID` to this so it's easy to replace with `Profile` during migration?

Reply:
I would say no, because that would be duplicated information. When you request the ProfileInformation by the userId you obviously already have the userId. I would rather prefer to stick as close to the API as possible than implementing convenient methods which could lead to inconsistent data (like profile.userId != theUserIdYouGaveIntoTheFunction)